### PR TITLE
Feature/fontcolor

### PR DESCRIPTION
**Pull request by ethanejohnsons**
This is part one of #1697. It adds color settings for the "Rotation", "Zoom", and "Roll" text on the main screen.

### CODE REVIEW

File: src/rendering/renderengine.cpp [hunk 1]
```diff
@@ -1148,8 +1168,8 @@ void RenderEngine::renderCameraInformation() {
         return;
     }
 
-    const glm::vec4 EnabledColor  = glm::vec4(0.2f, 0.75f, 0.2f, 1.f);
-    const glm::vec4 DisabledColor = glm::vec4(0.55f, 0.2f, 0.2f, 1.f);
+    const glm::vec4 EnabledColor  = _enabledFontColor.value(); //glm::vec4(0.2f, 0.75f, 0.2f, 1.f);
```
Comment (alexanderbock) [inline src/rendering/renderengine.cpp:1171]: Could remove the comment here, since it is no longer necessary.  Same below

**Review by alexanderbock — APPROVED** [review]
:+1:  Looks great!

**Comment by alexanderbock** [comment]
Sweet! Feel free to either merge or squash and merge. Your choice :)

Thanks for the help!

**Comment by ethanejohnsons** [comment]
I don't think I have the right permissions to merge. I'm fine if you wanna go ahead and merge it for me!

**Comment by alexanderbock** [comment]
👏

**Pull request merged**